CMakeLists.txt

```
@@ -1,34 +1,63 @@
 cmake_minimum_required(VERSION 3.17)
 cmake_policy(SET CMP0100 NEW)  # handle .hh files
-project(CLAP_HELPERS C CXX)
+project(clap-helpers C CXX)
 
-add_library(clap-helpers INTERFACE)
-target_include_directories(clap-helpers SYSTEM INTERFACE include)
-target_link_libraries(clap-helpers INTERFACE clap)
+option(CLAP_HELPERS_BUILD_TESTS "Build tests for the CLAP Helper" FALSE)
 
-# CLAP_BUILD_TESTS is inherited from clap
-if (${CLAP_BUILD_TESTS})
-    enable_testing()
-    find_package(Catch2 3 QUIET)
+add_library(${PROJECT_NAME} INTERFACE)
+target_include_directories(${PROJECT_NAME} INTERFACE include)
 
-    if (Catch2_FOUND)
-        add_executable(clap-helpers-tests EXCLUDE_FROM_ALL
+if (NOT TARGET clap)
+    message(STATUS "${PROJECT_NAME}: Downloading CLAP using CPM")
+
+    message(STATUS "${PROJECT_NAME}: including CPM")
+    include(cmake/external/CPM.cmake)
+    CPMAddPackage(
+            NAME "clap"
+            GITHUB_REPOSITORY "free-audio/clap"
+            GIT_TAG "next"
+            EXCLUDE_FROM_ALL TRUE
+            DOWNLOAD_ONLY TRUE
+            SOURCE_DIR cpm/clap
+    )
+
+    add_subdirectory(${CMAKE_BINARY_DIR}/cpm/clap)
+endif()
+target_link_libraries(${PROJECT_NAME} INTERFACE clap)
+
+if (${CLAP_HELPERS_BUILD_TESTS})
+    if (NOT DEFINED ${CLAP_HELPERS_TESTS_CXX_STANDARD})
```

I'm not completely sure about DEFINED syntax but in clap-wrapper the equivalent code is written like this:
```suggestion
    if (NOT DEFINED CLAP_HELPERS_TESTS_CXX_STANDARD)
```